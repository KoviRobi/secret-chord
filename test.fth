1 2 .s
over - .s
1+ dup rot .s
and .s
or .s
invert 1- .s
