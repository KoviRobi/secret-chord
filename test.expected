sp<2> 1 2
sp<2> 1 1
sp<3> 2 2 1
sp<2> 2 0
sp<1> 2
sp<1> -4
sp<1> -4
> sp<1> -4
